ci/unit/test_main.sh

```diff
@@ -7,6 +7,27 @@ set -ex
 # JENKINS-52781 tar function is breaking symlinks
 rm -rf test_results
 mkdir test_results
+
+# Check if this is a Bulleye stage
+USE_BULLSEYE=
+if [ -n "${STAGE_NAME:?}" ]; then
+  case $STAGE_NAME in
+    *Bullseye**)
+      USE_BULLSEYE="true"
+      ;;
+  esac
+fi
+
+: "${BULLSEYE:=}"
+
+if [ -n "$USE_BULLSEYE" ];then
```

Comment:
You can then use a proper boolean evaluation here:
```suggestion
if $USE_BULLSEYE; then
```

Reply:
Done